Library/Homebrew/formula.rb

```diff
@@ -1810,6 +1818,17 @@ def mirror(val)
       define_method(type) { |val| stable.send(type, val) }
     end
 
+    # @!attribute [w] gpg
+    # An optional mechanism to verify the download with GnuPG as well as the
+    # standard checksum verification. Upstream must publish signature files
+    # for us to be able to use this mechanism. Those signature files often
+    # have either a ".gpg" or ".sig" file extension.
+    #
+    # <pre>gpg "https://ftpmirror.gnu.org/wget/wget-1.18.tar.xz.sig"</pre>
+    def gpg(val)
+      stable.gpg(val)
+    end
```

Comment:
> Also, I'm interesting to know how do you indent to manage the trustiness of public key?

Verification _works_ without having to know a key in advance, but I was planning to improve this, even if not immediately immediately. It took me about two weeks to reconfigure the project after initial feedback & test those changes, so I've had to shove things together less coherently than initially desired or planned 😕.

```ruby
gpg "https://blah.com/blah-1.2.3.sig", :key => "0x872524DB9D74326C"
```
Or such.